.env

```diff
@@ -0,0 +1,7 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=um_db
+# MIGRATE_DIRECTION can be "up" for direct migration or "down" to reverse migration
```

Comment:
I suggest not to configure migration direction through environment until we need this feature. Our migrations will be always user as "up" in automated flow.

Reply:
Fix it